docs/4_4_1_can.adoc

```diff
@@ -607,35 +611,43 @@ If a Network FMU does not support wake up, this operation can be ignored on the
-The default value shall be `false` to allow the Network FMU to be run natively in each simulation scenario.
-If the Network FMU shall be configured to handle responses in the form of <<low-cut-can-confirm-operation, `Confirm-`>>, <<low-cut-can-bus-error-operation, `Bus Error-`>> and <<low-cut-can-arbitration-lost-operation, `Arbitration Lost`>> operations, the <<low-cut-can-bus-notification-parameter, `org.fmi_standard.fmi_ls_bus.Can_BusNotifications`>> parameter shall be set to `true`. +
+Therefore, a parameter with memberName=<<low-cut-can-bus-notification-parameter, `BusNotifications`>> can be added to a CAN Bus Terminal. +
+If a Network FMU supports bus notifications, the <<low-cut-can-bus-notification-parameter, `BusNotifications`>> parameter shall be exposed.
+The default value of this parameter shall be `false`. +
+_[The default value `false` allows a simple integration of Network FMUs to simulation scenarios where <<low-cut-can-confirm-operation, `Confirm-`>>, <<low-cut-can-bus-error-operation, `Bus Error-`>> or <<low-cut-can-arbitration-lost-operation, `Arbitration Lost`>> operations are not used.]_
+
+Only Network FMUs with the corresponding optionally exposed <<low-cut-can-bus-notification-parameter, `BusNotifications`>> parameter set to `true` might wait for <<low-cut-can-confirm-operation, `Confirm-`>>, <<low-cut-can-bus-error-operation, `Bus Error-`>> and <<low-cut-can-arbitration-lost-operation, `Arbitration Lost`>> operations and respond accordingly, Network FMUs must not wait otherwise ("fire-and-forget"). +
 _[This does not necessarily mean the FMU must fully support CAN error handling or sophisticated arbitration mechanisms._
 _A simple Network FMU might also choose to treat Bus Error or Arbitration operations in a simplified manner, e.g., by treating them as a positive confirmation.]_
 
-.FMU parameter for the configuration of bus notifications.
-[[figure-fmu-bus-notifications-parameter]]
+.Parameter to configure bus notifications within a CAN Bus Terminal of Network FMUs.
+[[figure-fmu--can-bus-notifications-parameter]]
 ----
-    org.fmi_standard.fmi_ls_bus.Can_BusNotifications
-        Description:  "Specifies whether the respective Network FMU can rely on a Confirm,
-                      Bus Error or Arbitration Lost operation after sending a message."
-        Type:         Boolean
-        Causality:    structuralParameter
-        Variability:  fixed
-        Start:        "false"
+ memberName:    BusNotifications
+ type:          Boolean
+ causality:     parameter
+ variability:   fixed
+ start:         "false"
 ----
 
-This structural parameter is mandatory for Network FMUs only.
-A Bus Simulation (FMU) does not require this structural parameter.
+A Bus Simulation FMU might indicate via a variable with memberName=<<low-cut-can-bus-notification-parameter, `BusNotifications`>> in the CAN Bus Terminals whether it sends bus notifications or not.
```

Comment:
@bmenne-dspace should we make this mandatory, by changing the might to shall?
I think it wouldn't be a big burden for Bus Simulation FMU exporters to always expose such a variable.

Reply:
I think optional is fine, because if its missing then its automatically "false". BTW: I think we should add the "to participate in an importer parameter propagation feature." sentence again

Reply:
@klausschuch @bmenne-dspace decided to make it mandatory. Note, this doesn't mean that it is mandatory to implement any behavior, just to express what is implemented.